libfwupd/fwupd-security-attr.c

```diff
@@ -61,6 +62,50 @@ fwupd_security_attr_flag_to_string (FwupdSecurityAttrFlags flag)
 	return NULL;
 }
 
+/**
+ * fwupd_security_attr_result_to_string:
+ * @result: A #FwupdSecurityAttrResult, e.g. %FWUPD_SECURITY_ATTR_RESULT_ENABLED
+ *
+ * Returns the printable string for the result enum.
+ *
+ * Returns: string, or %NULL
+ *
+ * Since: 1.5.0
+ **/
+const gchar *
+fwupd_security_attr_result_to_string (FwupdSecurityAttrResult result)
+{
+	if (result == FWUPD_SECURITY_ATTR_RESULT_VALID)
+		return "valid";
+	if (result == FWUPD_SECURITY_ATTR_RESULT_NOT_VALID)
+		return "not-valid";
```

Comment:
why not-valid instead of invalid?

Reply:
I was trying to keep the boolean states symmetric and possibly invertable in the future. I've used the "invalid" form in the user-visble text.

Reply:
ah OK.